app/src/lib/git/core.ts

```diff
@@ -230,6 +231,17 @@ export async function git(
 
   log.error(errorMessage.join('\n'))
 
+  if (gitError && gitError === DugiteError.PushWithFileSizeExceedingLimit) {
+    const result = getFileFromExceedsError(errorMessage.join())
+    if (result) {
+      const files = result.join('\n\t\t')
+
+      if (files && files !== '') {
+        gitResult.gitErrorDescription += '\n\nFiles:\n\t\t' + files
+      }
+    }
```

Comment:
Since `result` is always an array, this can probably be simplified like this:
```suggestion
    const files = result.join('\n\t\t')

    if (files !== '') {
      gitResult.gitErrorDescription += '\n\nFiles:\n\t\t' + files
    }
```